### PR TITLE
feat: twitter langchain actions tests

### PR DESCRIPTION
**Pull request by stat**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: twitter-langchain/tests/actions/test_account_details.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    account_details,
+)
+
+MOCK_ID = 1234
+MOCK_NAME = "Test Account"
+MOCK_USERNAME = "testaccount"
+
+def test_account_details_input_model_valid():
+    """Test that AccountDetailsInput accepts valid parameters."""
+    pass
```
Comment (rohan-agarwal-coinbase) [inline twitter-langchain/tests/actions/test_account_details.py:15]: is there a reason some of these tests are just `pass`?

**Review by rohan-agarwal-coinbase — CHANGES_REQUESTED** [review]

**Comment by cb-heimdall** [comment]
Review Error for rohan-agarwal-coinbase @ 2024-11-20 01:53:50 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

File: twitter-langchain/tests/actions/test_account_details.py (hunk 1 above)
Comment (stat) [reply twitter-langchain/tests/actions/test_account_details.py:15]: yeah, the model doesn't have any fields. thought i should at least have the test stubs in place

Comment (rohan-agarwal-coinbase) [reply twitter-langchain/tests/actions/test_account_details.py:15]: ah i see - i think we can delete this. i think it's useful to have test stubs when there's typically TODOs so it's a good way to not forget. but i think given the model doesn't have any fields and we don't intend on adding field, it's expected behavior and we can omit the tests

Comment (stat) [reply twitter-langchain/tests/actions/test_account_details.py:15]: roger that

**Review by rohan-agarwal-coinbase — APPROVED** [review]

**Pull request merged**